Umbraco-Uno/Working-With-Forms/Setting-up-a-form/index.md

```diff
@@ -0,0 +1,46 @@
+---
+versionFrom: 8.0.0
+---
+
+# Setting up a form
+
+In this guide, you will learn how to set up a form in the Umbraco Uno backoffice. 
+
+When you get your Uno project, it will not have any forms, so you will have to make them your self which can be a challenge when you have never done it before. In this guide, you will learn how to build a form from scratch.
```

Comment:
```suggestion
When you get your Uno project, it will not have any forms, so you will have to make them yourself. In this guide, you will learn how to build a form from scratch.
```